dockerfiles/hpcc/templates/thor.yaml

```diff
@@ -91,6 +91,9 @@ data:
 {{ toYaml (omit $thorScope "logging") | indent 6 }}
 {{- include "hpcc.generateLoggingConfig" (dict "root" $ "me" $thorScope) | indent 6 }}
     {{ $agentAppType }}:
+{{- if $thorScope.multiJobLinger }}
+      multiJobLinger: true
```

Comment:
is this the normal way of doing an option like this?  or 
multiJobLinger: {{ $thorScope.multiJobLinger | default false }}
?
(I don't know what is expected more commonly.)

Reply:
this is conditionally generating this property in another scope.
It's so that it appears in agent scope as well as (the original) thor scope, because it needs to be picked up at both ends.

It could unconditionally generate it as per your example, but I think it's ok to conditionally generate it only if present and true